src/classes/UTIL_RecordTypes_TEST.cls

```diff
@@ -35,7 +35,61 @@
 */
 
 @isTest
-private class UTIL_RecordTypes_TEST {
+public class UTIL_RecordTypes_TEST {
+    
+    /**
+    * @description Return a random valid record type for the sObject
+    * @param objectType The sObject type of the record type
+    * @return Id
+    */
+    public static Id getRandomRecordTypeId(sObjectType objectType) {
```

Comment:
- Helper methods should be moved to the end of the class, ie after unit tests.
- Code style convention: `sObjectType` > `SObjectType`
- Could you use any `UTIL_RecordTypes` method such as `getRecordTypeIdForGiftsTests` instead of the random record type Id method?
If not, let's make method name generic ie `getRecordTypeId(SObjectType objectType)`.